example/spec/testAPI/ExecutionStatusSpec.js

```diff
@@ -0,0 +1,153 @@
+'use strict';
+
+const { difference } = require('lodash');
+const fs = require('fs-extra');
+const { loadConfig } = require('../helpers/testUtils');
+const {
+  aws: { s3 },
+  stringUtils: { globalReplace },
+  testUtils: { randomStringFromRegex }
+} = require('@cumulus/common');
+const { createGranuleFiles } = require('../helpers/granuleUtils');
+const { buildAndExecuteWorkflow } = require('@cumulus/integration-tests');
+const config = loadConfig();
+const workflowName = 'IngestGranule';
+const granuleRegex = '^MOD09GQ\\.A[\\d]{7}\\.[\\w]{6}\\.006.[\\d]{13}$';
+const testDataGranuleId = 'MOD09GQ.A2016358.h13v04.006.2016360104606';
+const { api: apiTestUtils } = require('@cumulus/integration-tests');
+
+/**
+ * Set up files in the S3 data location for a new granule to use for this
+ * test. Use the input payload to determine which files are needed and
+ * return updated input with the new granule id.
+ *
+ * @param {string} bucket - data bucket
+ * @param {string} granuleId - granule id for the new files
+ * @param {string} inputPayloadJson - input payload as a JSON string
+ * @returns {Promise<Object>} - input payload as a JS object with the updated granule ids
+ */
+async function setupTestGranuleForAPI(bucket, granuleId, inputPayloadJson) {
```

Comment:
I think this should go into a helper function in `granuleUtils` since it's used by the ingest granule tests. Another option would be to put these tests in with the ingest granule tests so we don't run another execution of ingest granule.